lib/common/schemas.c

```diff
@@ -650,8 +652,53 @@ validate_xml(xmlNode *xml_blob, const char *validation, gboolean to_logs)
 }
 
 #if HAVE_LIBXSLT
+
+/* see crm_xml_error */
+static void
+crm_xslt_err(void *ctx, const char *msg, ...)
+G_GNUC_PRINTF(2, 3);
+
+static void
+crm_xslt_err(void *ctx, const char *msg, ...)
+{
+    int len = 0;
+    va_list args;
+    char *buf = NULL;
+    static int buffer_len = 0;
+    static char *buffer = NULL;
+
+    va_start(args, msg);
+    len = vasprintf(&buf, msg, args);
+
+    if (strchr(buf, '\n')) {
+        buf[len - 1] = 0;
+        if (buffer) {
+            crm_err("XSLT reports: %s%s", buffer, buf);
+            free(buffer);
+        } else {
+            crm_err("XSLT reports: %s", buf);
```

Comment:
I'm not sure about crm_xml_err(), but I think this function should log at warning severity instead of error, since we don't abort anything, we proceed with the transformation result.